homeassistant/components/epson/media_player.py

```diff
@@ -122,44 +125,81 @@ async def async_service_handler(service):
     hass.services.async_register(
```

Comment:
Don't register services like this. Instead use `platform.async_register_entity_service`. See https://github.com/home-assistant/core/blob/dev/homeassistant/components/sonos/media_player.py#L392-L400